maxim/logger/openai/async_completions.py

```
@@ -1,15 +1,75 @@
+    This class wraps an async OpenAI stream to automatically log generation results
+    and trace information when the stream is fully consumed. It accumulates chunks
+    as they are yielded and processes the complete response for logging when the
+    stream ends.
+    """
+
+    def __init__(
+        self,
+        stream: AsyncIterator[ChatCompletionChunk],
+        generation: Optional[Generation],
+        trace: Optional[Trace],
+        is_local_trace: bool,
```

Ruff (0.11.9)

Boolean-typed positional argument in function definition

(FBT001)